genomicFileHandler/read_info_extractor.py

```diff
@@ -242,21 +256,21 @@ def sam_info_PV4(vcf_object):
         pv4 = ( float(pv4_string[0]), float(pv4_string[1]), float(pv4_string[2]), float(pv4_string[3]) )
     else:
         pv4 = nan,nan,nan,nan
-    
+
     return pv4
-    
+
 
 
 ##### From Haplotype caller vcf:
 def haplo_MQ0(vcf_object):
     '''Total Mapping Quality Zero Reads'''
-    
+
     mq0 = vcf_object.get_info_value('MQ0')
     if mq0:
         mq0 = eval(mq0)
```

Comment:
Why `eval` instead of, say, `int`?

Reply:
Removing all haplo_XXX and sam_info_XXX functions. They were used to extract information from samtools and haplotype VCF files outputs, which were deprecated since version 2.